### PR TITLE
StyleGuideAction component DOM validates

### PR DESCRIPTION
**Pull request by slj**
Connects #1839 

Changing the P tag to DIV basically broke everything.  On further examination of the DOM, some cleanup was in order, including the removal of `<InlineForm />`.

My goal was to eliminate the console errors, but I think this component needs more work to be as reusable as we want.  But I think that's for another issue/ticket.

### CODE REVIEW

File: client/app/containers/StyleGuide/StyleGuideAction.jsx [hunk 1]
```diff
@@ -1,36 +1,29 @@
...
-            name="Submit End Product"
-          />
-        </div>
+  return <div>
+    <div className="cf-app cf-push-row cf-sg-layout cf-app-segment cf-app-segment--alt">
+    </div>
+    <div className="cf-app-segment" id="establish-claim-buttons">
+      <div className="cf-push-left">
+        <Button
+          name="Back To Preivew"
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideAction.jsx]: Also typo...and sentence case

File: client/app/containers/StyleGuide/StyleGuideAction.jsx [hunk 2]
```diff
@@ -1,36 +1,29 @@
...
+          classNames={['cf-btn-link']}
+        />
+      </div>
+      <div className="cf-push-right">
+        <Button
+          name="Cancel"
+          classNames={['cf-btn-link', 'cf-adjacent-buttons']}
+        />
+        <Button
+          name="Submit End Product"
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideAction.jsx]: can you also make this sentence case?

File: client/app/containers/StyleGuide/StyleGuideAction.jsx (hunk 1 above)
Comment (slj) [inline client/app/containers/StyleGuide/StyleGuideAction.jsx]: OMG thx for catching that

File: client/app/containers/StyleGuide/StyleGuideAction.jsx (hunk 2 above)
Comment (slj) [inline client/app/containers/StyleGuide/StyleGuideAction.jsx]: Yup

**Pull request merged**